apps/web/public/static/locales/en/common.json

```diff
@@ -2519,5 +2519,7 @@
   "outlook_connect_atom_already_connected_label": "Connected Outlook Calendar",
   "outlook_connect_atom_loading_label": "Checking Outlook Calendar",
   "booking_question_response_variables": "Booking question response variables",
+  "disable_org_url_label": "Disable Organization URL and redirect",
+  "disable_org_url_description": "Hide your deel.cal.com subdomain and redirect to deal.com instead.",
```

Comment:
```suggestion
  "disable_org_url_description": "Hide your deel.cal.com subdomain and redirect to deel.com instead.",
```

Reply:
will update

Reply:
@Amit91848 shouldn't we use acme.com as an example here?

Reply:
I thought so too acme would be better, but the screenshot in issue had deel in description.